pkg/client/kubernetes.go

```diff
@@ -0,0 +1,42 @@
+package client
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+	k8yaml "sigs.k8s.io/yaml"
+)
+
+type Client struct {
```

Comment:
Is there a better name for this? Maybe package `localk8s`?

Reply:
Can be renamed, not a fan of `LocalK8s` can be `KubernetesClient` or something